ml/src/main/scala/org/apache/spark/ml/h2o/algos/H2OAutoML.scala

```diff
@@ -209,12 +210,12 @@ trait H2OAutoMLParams extends Params {
   //
   // Param definitions
   //
-  private final val predictionCol = new NullableStringParam(this, "predictionCol", "Prediction column name")
+  private final val predictionCol = new Param[String](this, "predictionCol", "Prediction column name")
   private final val allStringColumnsToCategorical = new BooleanParam(this, "allStringColumnsToCategorical", "Transform all strings columns to categorical")
   private final val columnsToCategorical = new StringArrayParam(this, "columnsToCategorical", "List of columns to convert to categoricals before modelling")
   private final val ratio = new DoubleParam(this, "ratio", "Determines in which ratios split the dataset")
   private final val foldColumn = new NullableStringParam(this, "foldColumn", "Fold column name")
```

Comment:
Should this be renamed too then?

It seems there is a bunch of inconsistencies, eg.:
predictionCol
weightCol

vs

foldColumn
ignoredColumns

Reply:
Good point @michalkurka! We agreed with @jakubhava to slowly migrate parameters to *Col pattern, so that we will follow Spark naming conventions. Also it would make sense to rename ```predictionCol``` to ```labelCol``` as it's in Spark MLlib.

Reply:
@mn-mikke Okay, makes sense then! Thanks